### PR TITLE
Clean up demographics and ACS-based initialization

### PR DESCRIPTION
**Pull request by zmbc**
Continuing the cleanup of the PRL concept model document started in #1124. This PR focuses on the "demographics" section and the first component, which is everything we initialize from ACS. I want to separate the guardian part into its own component in the list, but am leaving that for a future PR.

### CODE REVIEW

File: docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst [hunk 1]
```diff
@@ -158,218 +158,181 @@ except in the case of some young simulants who are assigned :ref:`guardians <cen
...
+
+* White
+* Black
+* Latino
+* American Indian and Alaskan Native (AIAN)
+* Asian
+* Native Hawaiian and Other Pacific Islander (NHOPI)
+* Multiracial or Some Other Race
+  
+This composite indicator has the same categories used within IHME by the US Counties BoD team.
```
Comment (aflaxman) [inline docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst:295]: Are these the terms that this team is now using?

**Review by aflaxman — APPROVED** [review]

**Review by SylLutze — APPROVED** [review]

**Comment by aflaxman** [comment]
This is the paper that Laura says is the most recent example of this work:

Dwyer-Lindgren L, Kendrick P, Kelly YO, Sylte DO, Schmidt C, Blacker BF, et al. Life expectancy by county, race, and ethnicity in the USA, 2000–19: a systematic analysis of health disparities. The Lancet. 2022 Jul 2;400(10345):25–38.

File: docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst [hunk 2]
```diff
@@ -158,218 +158,181 @@ except in the case of some young simulants who are assigned :ref:`guardians <cen
...
+We have used single-state runs so far as a form of small-scale testing.
+We may want to generate final simulation outputs at the state level, or other smaller
+geographic areas, in the future; the specifics are to be determined.
+
+.. _census_prl_population_size:
+
+2.2.3 Population size
+~~~~~~~~~~~~~~~~~~~~~
+
+Our final data outputs will be at the scale of the real United States population.
```
[the remark this answers is not in the document]
Comment (zmbc) [reply docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst:198]: I'll add a TODO; I think this hasn't been decided yet.

**Comment by aflaxman** [comment]
Sounds good

**Pull request merged**